docs/source/getting_started/getting_started.md

```diff
@@ -66,9 +66,36 @@ Each document has a unique identifier that can be used to refer to it in the sys
 
 The `snippet` field is used to inform the system about the content of the document; it is used as input to Xaynia to generate a mathematical representation of the document that we can use to match similar documents.
 
-For this reason, it is essential that the snippet clearly represents the content of the document. In this case, we took a few representative sentences from the document and used them as a snippet. Since the amount of data that Xaynia can analyze is limited, if it is not possible to provide a concise snippet, we can use the per-document option `summarize`; when enabled, the system will summarise the content of the document to create a snippet. Alternatively the `split` option can be used which will split the document into multiple snippets,
+For this reason, it is essential that the snippet clearly represents the content of the document. In this case, we took a few representative sentences from the document and used them as a snippet. Since the amount of data that Xaynia can analyze is limited, if it is not possible to provide a concise snippet, we can use the per-document option `summarize`; when enabled, the system will summarize the content of the document to create a snippet. Alternatively the `split` option can be used which will split the document into multiple snippets,
```

Comment:
```suggestion
For this reason, it is essential that the snippet clearly represents the content of the document. In this case, we took a few representative sentences from the document and used them as a snippet. Since the amount of data that Xaynia can analyze is limited, we can use the per-document option `summarize` when it is not possible to provide a concise snippet; when enabled, the system will summarise the content of the document to create a snippet. Alternatively, the `split` option can be used which will split the document into multiple snippets,
```
in the rest of the documentation we stick with the british spelling , so i would keep `summarise`

personalisation